Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -113,14 +116,21 @@ def fetch_indicators_command(client: Client, indicator_type: str, feedTags: list
     :param feedTags: the indicator tags
     """
     indicators = []
-    for result in client.build_iterator(**kwargs):
+    feeds_results = []
+    for feed_name, feed in client.feed_name_to_config.items():
+        build_iterator_paging = feed.get('build_iterator_paging')
+        if build_iterator_paging:
+            feeds_results.append({feed_name: build_iterator_paging(client, feed, **kwargs)})
```

Comment:
We need to add error handling for that case since any function can be sent here.
Try to do that as informative to the user as it gets.